docs/ledger-lunie-guide.md

```diff
@@ -0,0 +1 @@
+# Using Lunie with the Ledger Nano S
```

Comment:
What's the point of this file? It should at least link to something...

Reply:
++. Please remove the file

Reply:
no point, I just forgot to delete it